3.1/bookworm/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # PLEASE DO NOT EDIT IT DIRECTLY.
 #
 
-ARG BASE_IMAGE=rstudio/r-base
+ARG BASE_IMAGE=posit/r-base
```

Comment:
@jmwoliver I missed this earlier too, but only the templates were updated before. Running `make update-all` just now got all the generated Dockerfiles updated too.

Reply:
Oh good catch. I thought I ran that, but I guess I didn't do it right. Thanks for updating it!